sycl/include/sycl/info/info_desc.hpp

```diff
@@ -193,14 +193,9 @@ template <typename T, T param> struct compatibility_param_traits {};
   } /*namespace info */                                                        \
   } /*namespace Namespace */
 
-namespace ext::oneapi::experimental {
-
-enum class graph_support_level { unsupported = 0, native = 1, emulated = 2 };
```

Comment:
Since it is experimental, it is fine to drop it without deprecation. That said, it wouldn't be the hardest to keep for maybe a release or two. @gmlueck | @stdale-intel - Preferences here?

Reply:
@steffenlarsen @gmlueck Thanks for your assessment and feedback! I also see no need to depreciate the feature with our experimental extension. As Greg pointed out it's an easy switch to the new aspect, so let's just drop it.